source/include/ota_appversion32.h

```diff
@@ -61,9 +62,9 @@ typedef struct
         #endif /* if ( defined( __BYTE_ORDER__ ) && defined( __ORDER_LITTLE_ENDIAN__ ) && __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__ ) || ( __little_endian__ == 1 ) || WIN32 || ( __BYTE_ORDER == __LITTLE_ENDIAN ) */
         uint32_t unsignedVersion32;
         int32_t signedVersion32;
-    } u; /*!< Version based on configuration in big endian or little endian. */
+    } u; /*!< @brief Version based on configuration in big endian or little endian. */
```

Comment:
In lines 44-64, does doxygen not require the variables to be `@brief`-ed? like the below one?
```
struct version
{
     uint16_t build;
     uint8_t minor;
     uint8_t major;
} x;
```

Even if that is not required, I think we should add it just for completeness.

Reply:
Done: https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/116/commits/3f19470223e963500c4fc69e1756bb21292903a9